rust/automerge-c/README.md

```diff
@@ -5,6 +5,14 @@ for other language bindings that have good support for calling C functions.
 
 # Installing
 
+## Prerequisites
+
+* Cargo >= 1.71.0
+* CMake >= 3.25
+* CMocka >= 1.1.5
+* Doxygen >= 1.9.1
```

Comment:
Doxygen is only required to build the docs, right? I tried to reflect that in #1003; the distinction is lost here.